docs/zkapps/tutorials/06-offchain-storage.mdx

```diff
@@ -210,7 +210,7 @@ Continuing, let's setup our smart contract:
 
 Here we add 3 pieces of state to our contract: the public key of the storage server, the "storageNumber"—which is used to ensure the storage server is actively storing states, and the root of the merkle tree.
 
-We also initialize our zkApp's initialize for these three values, by setting them to the public key of our storage server, setting storage number to 0, and storing the root of an empty tree.
+We also initialize our zkApp's initState function for these three values, by setting them to the public key of our storage server, setting storage number to 0, and storing the root of an empty tree.
```

Comment:
I think this would be a better sentence:
"We also initialize our zkApp's **state** for these three values ..."

Reply:
noted ! have updated per your feedback 😃